Dockerfile.ci

```diff
@@ -216,7 +216,7 @@ RUN mkdir -p /opt/bats/lib/bats-file \
 RUN echo "export PATH=/opt/bats/bin:${PATH}" >> /root/.bashrc
 
 # Additional scripts for managing BATS addons
-COPY scripts/ci/dockerfiles/bats/load.bash /opt/bats/lib/
```

Comment:
Should we remove `scripts/ci/dockerfiles/bats/load.bash` then?

Reply:
No. It is needed in the "local" bats. It's a bit of duplication (but very little) but it helps to keep stuff in order. The scripts in "ci" are those that are used in the host, where the scripts in "docker" are needed by the Docker image to be used as part of the build context.

The problem is that in the workflow run the "ci" scripts for security come always from master (so that no-one can abuse the GITHUB_TOKEN with write access, but the Dockerfile.ci and all the stuff that is needed to build the Docker image needs to come from the PR. 

I know it's not straightforward :( . But this way we have huge optimisations when building the image only once in workflow_run,

Reply:
I've added a bit more explanation on that in the commit